pages/home.vue

```diff
@@ -0,0 +1,296 @@
+          nameGroup: this.selectedNameGroup,
+          nameClient: this.selectedNameClient,
+          name: this.selectedName,
+          userId: this.$auth.user.id,
+          role: this.$auth.user.roleName,
+          token: this.$auth.getToken('local'),
+          nbPerPage: this.nbPerPage,
+          lastId: 2147483647
+        }
+
+        let gallery = await axiosGet({
+          axios: this.$axios,
+          url: '/api/mission/galery',
+          params
+        })
+
+        if (gallery.data.mission) {
+          this.nbRqt = 1
+
+          this.missions = gallery.data.mission[0]
+
+          if (this.missions && this.missions.length > 0) {
+            this.lastId = this.missions[this.missions.length - 1].id
+          } else {
+            this.lastId = 2147483647
+          }
+        }
+      },
+      scroll() {
+        window.onscroll = () => {
+
+          if (this.missions.length === this.nbRqt * this.nbPerPage) {
+            let bottomOfWindow = document.documentElement.scrollTop + window.innerHeight === document.documentElement.offsetHeight
+
+            if (bottomOfWindow) {
+
+              const params = {
+                userId: this.$auth.user.id,
+                role: this.$auth.user.roleName,
+                token: this.$auth.getToken('local'),
+                nbPerPage: this.nbPerPage,
+                lastId: this.lastId
+              }
+
+              this.getGallery(params)
+            }
+          }
+        }
+      },
+      async getGallery(params) {
+        params.nbPerPage = params.nbPerPage ? params.nbPerPage : this.nbPerPage
+        params.lastId = params.lastId ? params.lastId : 2147483647
+
+        let gallery = await axiosGet({
+          axios: this.$axios,
+          url: '/api/mission/galery',
+          params
+        })
+        if (gallery.data.mission && gallery.data.mission[0].length > 0) {
+          this.nbRqt++
+
+          this.missions = this.missions.concat(gallery.data.mission[0])
+
+          if (this.missions && this.missions.length > 0) {
+            this.lastId = this.missions[this.missions.length - 1].id
+          } else {
+            this.lastId = 2147483647
+          }
+        }
+      },
+    }
+  }
+</script>
+
+<style scoped>
+    #page {
+        display:flex;
+        flex-direction: column;
+        justify-content: center;
+        margin: 1em auto;
+        width: 90%;
+    }
+    #drop a {
+        text-decoration: none;
+        color: black;
+    }
+    #content {
+        width: 100%;
+        margin: 1em auto;
+        display: flex;
+        justify-content: space-evenly;
+        flex-flow: wrap;
+    }
+    .link-btn a {
+        color: #FFF;
+    }
+    .card-big {
+        height: 360px;
+    }
+    .card-small {
+        height: 320px;
+    }
+    .card-body {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: flex-end;
+    }
+    /* .card-image-top {
```

Comment:
Will it be use later or can we delete this ?

Reply:
Good catch .

Reply:
If you don't know if it's useful , it's not useful.